Configurations/Sanitizers.xcconfig

```diff
@@ -116,7 +116,7 @@ WK_LIBFUZZER_OTHER_LDFLAGS_YES = $(WK_SANITIZER_COVERAGE_OTHER_FLAGS);
 // Fuzzilli for JavaScriptCore
 
 WK_FUZZILLI_OTHER_CFLAGS_YES = $(WK_SANITIZER_COVERAGE_OTHER_FLAGS) -DENABLE_FUZZILLI=1;
-WK_FUZZILLI_OTHER_LDFLAGS_YES = $(WK_SANITIZER_COVERAGE_OTHER_FLAGS);
+WK_FUZZILLI_OTHER_LDFLAGS_YES = $(WK_SANITIZER_COVERAGE_OTHER_FLAGS) -Xlinker -no_warn_inits -Xlinker -not_for_dyld_shared_cache;
 
```

Comment:
What is the reason of this error? Can you add comment here about that?

Reply:
I believe the linker defaults to linking with `-no_inits` in more recent (internal) SDKs for projects that are part of the dyld shared cache, since `-no_inits` is not see anywhere on the command-line (which also means that `JavaScriptCore.xcconfig` was not setting the command-line switch).

The Fuzzilli configuration hadn't been built with an SDK since that change, so we didn't know about it until recently.

The fix limits setting these linker switches to when `ENABLE_FUZZILLI=YES`.

Reply:
> OK, r=me with adding these descriptions here as a comment.

I'm a little unclear on the "ask" here, but it sounds like you want me to put the above text in the commit message, so I'll do that before landing.